samples/client/petstore/python/tests/test_deserialization.py

```diff
@@ -1,241 +0,0 @@
-# coding: utf-8
```

Comment:
Please add all files back in at samples/client/petstore/python/tests
those tests were manually written and are needed

Reply:
Done

Reply:
Can you resolve this conversation?